tests/cursor_test.go

```diff
@@ -29,7 +29,7 @@ import (
 	"time"
 
 	"github.com/dchest/uniuri"
-	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
```

Comment:
In this file you switch from "assert" to "require", in all the others not. Why is that?

Reply:
In the cases where `require` is used, other tests follow it.
I don't want to run these following tests when the requirement has failed